src/api/index.js

```diff
@@ -184,6 +184,488 @@ const COUNTRIES_LOCATION = {
     zoom: '3',
     center: '8.7832,34.5085',
     twitterHandle: '@followlasg',
+  }
+};
+
+const COUNTIES_LOCATION = {
+  kenya: {
+    slug: 'kenya',
+    city: 'nairobi',
+    latitude: '0.17666667',
+    longitude: '37.90832778',
+    zoom: '6',
+    country:"kenya",
+    center: '0.17666667,37.90832778',
+  },
```

Comment:
1. Kenya is not a county,
2. It's missing `label` prop,
3. Have we deleted `COUNTRIES_LOCATION` or are we using it anywhere?